extensions/podman/src/compatibility-mode.ts

```diff
@@ -138,7 +138,7 @@ export class DarwinSocketCompatibility extends SocketCompatibility {
 
     // Prompt the user to restart the podman machine if it's running
     const isRunning = await findRunningMachine();
-    if (isRunning !== '') {
+    if (isRunning) {
```

Comment:
This is code duplication.

```
    const isRunning = await findRunningMachine();
    if (isRunning) {
      await this.promptRestart(isRunning);
    }

    return Promise.resolve();
```

Let's move check for machine name inside of promptRestart method.

After that we can rewrite the method as

```
  async enable(): Promise<void> {
    await this.runCommand('install', 'enabled')

    // Prompt the user to restart the podman machine if it's running
    return findRunningMachine().then(machine => this.promptRestart(machine));
  }
```